apps/vercel/src/connectors/users.ts

```diff
@@ -0,0 +1,47 @@
+import { env } from '@/env';
+import { VercelError } from './commons/error';
+
+
+export type VercelUser = {
+  role: string;
+  uid: string;
+  name: string;
```

Comment:
```suggestion
  name?: string;
```

Tried it and sometimes name is undefined